c2corg_ui/static/js/map/map.js

```diff
@@ -973,6 +974,20 @@ app.MapController.prototype.simplifyFeature_ = function(feature) {
 /**
  * @export
  */
+app.MapController.prototype.toggleFullscreen = function() {
+  this.isFullscreen = !this.isFullscreen;
+  this.scope_.isFullscreen = this.isFullscreen;
```

Comment:
Are you sure you need ``this.scope_.isFullscreen``? In ``c2corg_ui/static/partials/map/map.html`` you use ``mapCtrl.isFullscreen`` which is ``this.isFullscreen``, not a scope property.

Reply:
Obviously not, in dev it's not working without that, i dont know why, Maybe i had an other error.
It's fixed :)